README.md

```diff
@@ -242,6 +242,7 @@ a Python version equal or greater than 3.5 or 2.7.
 * **Numpydoc**: Used by Jedi to get function return types from Numpydocstrings.
 * **Cloudpickle**: Serialize variables in the IPython kernel to send to Spyder.
 * **spyder-kernels** 1.4.0+: Jupyter kernels for the Spyder console.
+* **jupyter_client** 5.2.5+: Client for Jupyter kernels.
```

Comment:
Please move this dependency to spyder-kernels (which already has a minimal required version of jupyter_client).